README.md

```diff
@@ -23,6 +23,19 @@ Beaker functionality has been extended through the use of libraries available as
 
 See [LICENSE](LICENSE) file.
 
-#Support
+#Support & Issues
 
 Please log tickets and issues at our [Beaker Issue Tracker](https://tickets.puppetlabs.com/issues/?jql=project%20%3D%20BKR).  In addition there is an active #puppet-dev channel on Freenode.
+
+For additional information on filing tickets, please check out our [CONTRIBUTOR doc](CONTRIBUTING.md),
+and for ticket lifecycle information, checkout our [ticket process doc](docs/meta/ticket_process.md).
+
+#Maintainers
+
+Beaker is maintained by Puppet's Quality Engineering (QE) Team. In particular, our current Beaker maintainers are:
+
+| Name           | Email                |
+|:--------------:|:--------------------:|
+| Kevin Imber    | <ki@puppet.com>      |
+| Tony Vu        | <tony.vu@puppet.com> |
+| Sarah Thompson | <sarah@puppet.com>   |
```

Comment:
looks good.  only thing we did in addition, on rototiller, for instance, is to add the catch-all qa-team@ at the end.

not sure if that's advisable or not 
![spam](http://vignette2.wikia.nocookie.net/uncyclopedia/images/d/d0/Spam.gif/revision/20070303152602)!